certvalidator/validate.py

```diff
@@ -1,7 +1,8 @@
 # coding: utf-8
 from __future__ import unicode_literals, division, absolute_import, print_function
-
+from datetime import datetime
```

Comment:
This can be removed since we will always have a `revocation_moment` on `validation_context`.